app/graphql/resolvers/surveys_resolver.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Resolvers
+  class SurveysResolver < BaseResolver
+    description 'Survey list'
+    type Types::SurveyType.connection_type, null: false
```

Comment:
What does this do? I am also curious of `connection_type_class Types::Base::Connection` in a file below. Why is this needed? Thanks for enlightening me in advance 🙇

Reply:
Nevermind I saw the link to https://graphql-ruby.org/pagination/using_connections.html